src/Akka.HealthCheck.Tests/Transports/SocketStatusTransportSpecs.cs

```diff
@@ -41,14 +42,13 @@ public async Task Should_successfully_open_and_close_signal()
             var tcpClient = new TcpClient(AddressFamily.InterNetwork);
             await tcpClient.ConnectAsync(IPAddress.Loopback, PortNumber);
 
+            await AwaitAssertAsync(async () => (await tcpClient.GetStream().ReadAsync(new byte[10], 0, 10)).Should().Be(8));
+
             var deleteResult = await Transport.Stop(null, CancellationToken.None);
             deleteResult.Success.Should().BeTrue();
 
-            await AwaitAssertAsync(async () => 
```

Comment:
Moving this to before the `Transport.Stop()` call. `Transport.Stop()` close and disposes the socket, reading the tcp client buffer here does not make any sense.